pkgs/development/python-modules/emv/default.nix

```
@@ -0,0 +1,37 @@
+{ stdenv, buildPythonPackage, fetchFromGitHub
+, click, enum-compat, pyscard, pycountry, terminaltables
+, pytestCheckHook, unittest2
+}:
+
+buildPythonPackage rec {
+  pname = "emv";
+  version = "1.0.9";
+
+  src = fetchFromGitHub {
+    owner = "russss";
+    repo = "python-emv";
+    rev = "v${version}";
+    hash = "sha256:1ldibcy1wgnaimf8ri79lg8im1mhfqajrigfflvizw0hb0ivh8al";
+  };
+
+  checkInputs = [ pytestCheckHook unittest2 ];
+  propagatedBuildInputs = [
+    enum-compat
+    click
+    pyscard
+    pycountry
+    terminaltables
+  ];
+
+  # argparse is part of the standardlib
+  prePatch = ''
+    substituteInPlace setup.py --replace '"argparse",' ""
+  '';
+
+  meta = with stdenv.lib; {
+    homepage = "https://github.com/russss/python-emv";
+    description = "A Python implementation of the EMV smartcard protocol, used for chip-and-PIN payments";
```

Should be ~80 characters or less.
```suggestion
    description = "Python implementation of the EMV smartcard protocol";
```

---

Reworded. Dropped the "Python" as well, since this is in pythonPackages so it's obvious.